packages/text-block/src/getPlugins.tsx

```diff
@@ -0,0 +1,56 @@
+/* (c) Copyright Frontify Ltd., all rights reserved. */
+
+import {
+    AlignCenterPlugin,
+    AlignJustifyPlugin,
+    AlignLeftPlugin,
+    AlignRightPlugin,
+    BoldPlugin,
+    BreakAfterPlugin,
+    ButtonPlugin,
+    CheckboxListPlugin,
+    CodePlugin,
+    EmojiPlugin,
+    ItalicPlugin,
+    LinkPlugin,
+    OrderedListPlugin,
+    ParagraphPlugin,
+    PluginComposer,
+    ResetFormattingPlugin,
+    StrikethroughPlugin,
+    TextStylePlugin,
+    UnderlinePlugin,
+    UnorderedListPlugin,
+} from '@frontify/fondue';
+
+export const getPlugins = (columns?: number, gap?: number) => {
```

Comment:
gap doesn't match the type used in the plugin

Reply:
Thanks - changed!